test/quantization.cpp

```diff
@@ -782,8 +765,7 @@ TEST_CASE(dot_half_1arg)
                                    zp_b);
         auto qb  = mm->add_instruction(migraphx::make_op("quantizelinear"), x, scale_b, zp_b);
         auto dqb = mm->add_instruction(migraphx::make_op("dequantizelinear"), qb, scale_b, zp_b);
-        auto r   = mm->add_instruction(
-            migraphx::make_op("dot", {{"alpha", 1.0f}, {"beta", 0.0f}}), dqa, dqb);
+        auto r   = migraphx::add_apply_alpha_beta<float>(*mm, {dqa, dqb}, "dot");
```

Comment:
Same as above, no need to call this function.

Reply:
done